SConstruct

```diff
@@ -866,10 +873,10 @@ except subprocess.CalledProcessError:
     env["git_commit"] = "unknown"
 
 # Print values of all build options:
-print("Configuration variables read from 'cantera.conf' and command line:")
-for line in open('cantera.conf'):
-    print('   ', line.strip())
-print()
+msg = ["Configuration variables read from 'cantera.conf' and command line:"]
+for line in cantera_conf:
+    msg.append(f"    {line}")
+logger.info("\n".join(msg))
```

Comment:
```suggestion
cantera_conf = Path("cantera.conf").read_text()
logger.info("Configuration variables read from 'cantera.conf' and command line:")
logger.info(cantera_conf)
```

Reply:
Could also do `textwrap.indent(cantera.conf, '    ')` to retain the previous behavior.